internal/namespaces/baremetal/v1alpha1/custom_server.go

```diff
@@ -0,0 +1,99 @@
+package baremetal
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	serverActionTimeout = 20 * time.Minute
```

Comment:
is this enough ? how sure are we that the test will complete in the given timeframe ?

Reply:
We are never sure about server delivery. So far, 20 minutes are enough. We will bump it if required